runtime/manta-pc/src/lib.rs

```diff
@@ -48,7 +48,7 @@ pub use sp_runtime::BuildStorage;
 use pallet_xcm::{EnsureXcm, IsMajorityOfBody, XcmPassthrough};
 use polkadot_parachain::primitives::Sibling;
 use polkadot_runtime_common::{BlockHashCount, RocksDbWeight, SlowAdjustingFeeUpdate};
-use xcm::v0::{BodyId, Junction, MultiLocation, NetworkId};
+use xcm::latest::prelude::*;
```

Comment:
Can we find a way to avoid using `latest`?
Also, can we not use `*`?

Reply:
This seems to be fine actually. Here's all that is included:
https://github.com/paritytech/polkadot/blob/release-v0.9.10/xcm/src/v1/mod.rs#L52-L81
And these modules are all private so i can't include them directly.